frontend/src/pages/functional_components/RecurringRunDetailsV2FC.test.tsx

```diff
@@ -0,0 +1,370 @@
+    fullTestV2RecurringRun.experiment_id = 'test-experiment-id';
+    TestUtils.makeErrorResponseOnce(getExperimentSpy, 'woops!');
+    render(
+      <CommonTestWrapper>
+        <RecurringRunDetailsRouter {...generateProps()} />
+      </CommonTestWrapper>,
+    );
+    await waitFor(() => {
+      expect(getRecurringRunSpy).toHaveBeenCalled();
+    });
+
+    expect(updateBannerSpy).toHaveBeenLastCalledWith(
+      expect.objectContaining({
+        additionalInfo: 'woops!',
+        message: `Error: failed to retrieve this recurring run's experiment. Click Details for more information.`,
+        mode: 'warning',
+      }),
+    );
+
+    // "Still loads run" means that the details are still rendered successfully.
+    screen.getByText('Enabled');
+    screen.getByText('Yes');
+    screen.getByText('Trigger');
+    screen.getByText('Every 1 hours');
+    screen.getByText('Max. concurrent runs');
+    screen.getByText('50');
+    screen.getByText('Catchup');
+    screen.getByText('false');
+    screen.getByText('param1');
+    screen.getByText('value1');
+  });
+
+  it('shows top bar buttons', async () => {
+    render(
+      <CommonTestWrapper>
+        <RecurringRunDetailsRouter {...generateProps()} />
+      </CommonTestWrapper>,
+    );
+    await waitFor(() => {
+      expect(getRecurringRunSpy).toHaveBeenCalled();
+      expect(updateToolbarSpy).toHaveBeenCalledWith(
+        expect.objectContaining({
+          actions: expect.objectContaining({
+            cloneRecurringRun: expect.objectContaining({ title: 'Clone recurring run' }),
+            refresh: expect.objectContaining({ title: 'Refresh' }),
+            enableRecurringRun: expect.objectContaining({ title: 'Enable', disabled: true }),
+            disableRecurringRun: expect.objectContaining({ title: 'Disable', disabled: false }),
+            deleteRun: expect.objectContaining({ title: 'Delete' }),
+          }),
+        }),
+      );
+    });
+  });
+
+  it('enables Enable buttons if the run is disabled', async () => {
+    fullTestV2RecurringRun.status = V2beta1RecurringRunStatus.DISABLED;
+    render(
+      <CommonTestWrapper>
+        <RecurringRunDetailsRouter {...generateProps()} />
+      </CommonTestWrapper>,
+    );
+    await waitFor(() => {
+      expect(getRecurringRunSpy).toHaveBeenCalled();
+      expect(updateToolbarSpy).toHaveBeenCalledWith(
+        expect.objectContaining({
+          actions: expect.objectContaining({
+            cloneRecurringRun: expect.objectContaining({ title: 'Clone recurring run' }),
+            refresh: expect.objectContaining({ title: 'Refresh' }),
+            enableRecurringRun: expect.objectContaining({ title: 'Enable', disabled: false }),
+            disableRecurringRun: expect.objectContaining({ title: 'Disable', disabled: true }),
+            deleteRun: expect.objectContaining({ title: 'Delete' }),
+          }),
+        }),
+      );
+    });
```

Comment:
Can we add a test for checking the resulting update of the page after the Enable button is clicked?

```
    screen.getByText('Enabled');
    screen.getByText('No');

    mock the new "get recurring run" response to return yes for "enabled" field
    click enable button
    wait for the response

    screen.getByText('Enabled');
    screen.getByText('Yes');
 ```